ios/AppReviewModule.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  AppReviewModule.swift
+//  ItaliaApp
+//
+//  Created by Cristiano Tofani on 19/03/25.
+//
+
+import Foundation
+import UIKit
+import StoreKit
+
+
+/** `@objc` attribute exposes Swift methods to the Objective-C runtime**/
+@objc(AppReviewModule)
+class AppReviewModule: NSObject {
+  
+  @objc
+  func requestReview () -> Void {
+    if #available(iOS 14.0, *) {
+      let activeWindowScene = UIApplication.shared.connectedScenes.filter { scene in
+        return scene.activationState == .foregroundActive && scene is UIWindowScene
+      }.first
+      
+      if let scene = activeWindowScene as? UIWindowScene {
+        SKStoreReviewController.requestReview(in: scene)
```

Comment:
I noticed we're using [SKStoreReviewController](https://developer.apple.com/documentation/storekit/skstorereviewcontroller). Isn't it deprecated in favor of [RequestReviewAction](https://developer.apple.com/documentation/storekit/requestreviewaction) on iOS 16+? Should we consider adopting the new API, possibly with a version check? What do you think?